src/types/onyx/OriginalMessage.ts

```diff
@@ -270,20 +270,134 @@ type OriginalMessageChangeLog = {
     /** When was it last modified */
     lastModified?: string;
 
-    /** New role of user */
-    newValue?: string;
+    /** New role of user or new value of the category/tag field */
+    newValue?: boolean | string;
 
-    /** Old role of user */
-    oldValue?: string;
+    /** Old role of user or old value of the category/tag field */
+    oldValue?: boolean | string;
```

Comment:
@Krishna2323 why can this be boolean? Maybe you can give an example where it'll be boolean, and string.

Reply:
interesting..
why don't we have a new key for this? As a user of this field, I will have to type check every time and might not know how to handle it.